Documentation/gettingstarted/kind.rst

```diff
@@ -54,9 +54,19 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v1
-   controllers ``net_cls`` and ``net_prio`` have to be disabled, or cgroup v1 has
-   to be disabled (e.g. by setting the kernel ``cgroup_no_v1="all"`` parameter).
+   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v2
+   needs to be enabled, and cgroup v1 controllers ``net_cls`` and ``net_prio``
+   have to be disabled, or cgroup v1 has to be disabled (e.g. by setting the kernel ``cgroup_no_v1="all"`` parameter).
+   This ensures that Kind nodes have their own cgroup namespace, and cilium can
+   attach BPF programs at the right cgroup hierarchy. To verify this, run the
+   following commands on the host, and check if the output values are different
+
+   .. code-block:: shell-session
+
+      $ sudo ls -al /proc/`docker inspect -f '{{.State.Pid}}' kind-control-plane`/ns/cgroup
+      $ sudo ls -al /proc/self/ns/cgroup
+
+   Follow this `Pull Request <https://github.com/cilium/cilium/pull/16259>`__ for more details.
```

Comment:
```suggestion
   Refer to this `Pull Request <https://github.com/cilium/cilium/pull/16259>`__ for more details.
```
(Or just “See”?)
“Follow” sounds a bit odd to me, it reads like the PR is currently in progress (but it's merged).